.cirrus.yml

```
@@ -54,6 +57,20 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
 #  install_script:
 #    - choco install python --version=3.7.7 -y
 
+lint_task:
```

```suggestion
task:
```

Can be shorter (because you provided a `name:` )